pycfmodel/model/resources/security_group_ingress.py

```diff
@@ -47,3 +50,9 @@ def ipv4_slash_zero(self) -> bool:
 
     def ipv6_slash_zero(self) -> bool:
         return self.Properties.ipv6_slash_zero()
+
+
+class RDSDBSecurityGroupIngress(Resource):
+    TYPE_VALUE: ClassVar = "AWS::RDS::DBSecurityGroupIngress"
+    Type: str = TYPE_VALUE
+    Properties: DBSecurityGroupIngressProp
```

Comment:
Careful, with this, we will miss this parameter [DBSecurityGroupName](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-security-group-ingress.html#cfn-rds-securitygroup-ingress-dbsecuritygroupname).
And we won't be able to model the resource

Reply:
added